scripts/run_e2e_tests.sh

```diff
@@ -86,6 +86,9 @@ fi
 set -e
 source $(dirname $0)/setup.sh || exit 1
 source $(dirname $0)/setup_gae.sh || exit 1
+if [ "$TRAVIS" == true ]; then
```

Comment:
I think you'll have to enclose true within single quotes here and below: `'true'`, similar to the comparisons we have in the scripts section of .travis.yml?

Reply:
Also from this answer : https://stackoverflow.com/a/21768067
Looks like the correct usage is  
```
if $TRAVIS; then
...
```
Am I reading it correctly?

Reply:
Okay, from what I have understood, I think there are a number of ways of doing this. In order to be consistent throughout, let's use: `if [ "$TRAVIS" == 'true' ];` provided that works fine, since we're using the same in .travis.yml.